contrib/Dockerfile.manylinux

```
@@ -321,6 +298,29 @@ RUN export UV_INDEX="https://download.pytorch.org/whl/cu$(echo $CUDA_VERSION | c
 # Upgrade setuptools to latest version for compatibility with PEP 639 (license format)
 RUN uv pip install --upgrade 'setuptools>=80.9.0'
 
+ARG UCX_REF="v1.21.x"
+RUN cd /usr/local/src && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)